saleor/dashboard/product/forms.py

```diff
@@ -14,6 +16,18 @@
 from .widgets import ImagePreviewWidget
 
 
+class RichTextField(forms.CharField):
+    """A field for rich text editor, providing backend sanitization."""
+
+    widget = RichTextEditorWidget
+    ALLOWED_TAGS = BLEACH_ALLOWED_TAGS + ['br', 'p', 'h2', 'h3']
```

Comment:
Could we turn this into a setting for edge situations when there needs to be an iframe to offsite product details or h4/5 or partner's tracking script?

Reply:
I chose tags based on usage of our widget. I'm not sure if we need make it configurable, as in edge cases you can always inherit the field and overwrite `ALLOWED_TAGS`.